conf/quartz.properties

```diff
@@ -0,0 +1,450 @@
+# Copyright 2008 - 2017 Hitachi Vantara.  All rights reserved. 
```

Comment:
Is this file used in any way to build the Pentaho container image? If not, please remove it from git.